esp-hal/src/dma/mod.rs

```diff
@@ -709,6 +709,35 @@ macro_rules! dma_tx_buffer {
     }};
 }
 
+/// Convenience macro to create a [DmaRxStreamBuf] from buffer size and
+/// optional chunk size (uses max if unspecified).
+/// The buffer and descriptors are statically allocated and
+/// used to create the [DmaRxStreamBuf].
+///
+/// Smaller chunk sizes are recommended for lower latency.
+///
+/// ## Usage
+/// ```rust,no_run
+#[doc = crate::before_snippet!()]
+/// use esp_hal::dma_rx_stream_buffer;
+///
+/// let buf = dma_rx_stream_buffer!(32000);
+/// let buf = dma_rx_stream_buffer!(32000, chunk_size = 1000);
+/// # }
+/// ```
+#[macro_export]
+macro_rules! dma_rx_stream_buffer {
+    ($rx_size:expr) => {
+        dma_rx_stream_buffer!($rx_size, chunk_size = 4095)
+    };
+    ($rx_size:expr, chunk_size = $chunk_size:expr) => {{
```

Comment:
We should either modify all the macros to `chunk_size =` where applies, or this one should just take two integers. I'm not opposed to either solution, but I'd prefer avoiding mixing two approaches.

Reply:
Fair, I'll remove `chunk_size = ` for now. In a separate PR we can evaluate what is more readable for users